crates/rustc_codegen_spirv/src/builder/builder_methods.rs

```diff
@@ -782,22 +782,24 @@ impl<'a, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'tcx> {
         result
     }
 
+    // rustc has the concept of an immediate vs. memory type - bools are compiled to LLVM bools as
+    // immediates, but if they're behind a pointer, they're compiled to u8. The reason for this is
+    // because LLVM is bad at bools behind pointers (something something u1 bitmasking on load).
+    //
+    // SPIR-V allows bools behind *some* pointers, and disallows others - specifically, it allows
+    // bools behind the storage classes Workgroup, CrossWorkgroup, Private, Function, Input, and
+    // Output. In other words, "stuff the CPU can't see, bools are OK, stuff the CPU can't see, no
+    // bools". So, we always compile bools to bools, even if they're behind a pointer, and error if
```

Comment:
Should this be:

```suggestion
    // Output. In other words, "stuff the CPU can't see, bools are OK, stuff the CPU can see, no
    // bools". So, we always compile bools to bools, even if they're behind a pointer, and error if
```

?

Reply:
The sentence doesn't quite parse for me